core/server/admin/app.js

```diff
@@ -38,7 +38,7 @@ module.exports = function setupAdminApp() {
     // @TODO ensure this gets a local 404 error handler
     adminApp.use('/assets', serveStatic(
         config.get('paths').clientAssets,
-        {maxAge: utils.ONE_YEAR_MS, fallthrough: false}
+        {maxAge: (config.get('env') === 'production') ? utils.ONE_YEAR_MS : 0, fallthrough: false}
```

Comment:
I thought about making this configurable via config - but as the admin app is really an integral part of Ghost itself (as in people can't edit or re-upload the admin client), it seemed unnecessary.